audio_service_platform_interface/lib/utils.dart

```diff
@@ -0,0 +1,4 @@
+part of 'package:audio_service_platform_interface/audio_service_platform_interface.dart';
```

Comment:
made both files a part of this library

Reply:
perhaps we should come to this folder structure https://github.com/flutter/plugins/tree/master/packages/connectivity/connectivity_platform_interface/lib

Reply:
Not right now.... We can always come back to it later if it's an internal change that doesn't affect the API presented to apps.

Reply:
Let's keep this PR focused on the lints only.

Reply:
you are okay with the parts at the moment, right? i made it to make `_castMap` available in both files

Reply:
I'd like to avoid restructuring files for now. If necessary, we can replace `_castMap` by direct `cast` calls.

Reply:
how do i restructuring?

Reply:
hmm, with this it's not possible to import 'audio_service_platform_interface/method_channel_audio_service.dart'

i would hate to replace it as direct `cast`, but i think i will just copy it across two files

Reply:
I mean any changes to the file directory structure. @hacker1024 also made a PR with changes to the directory structure and these will be very difficult to merge, so I think it will be best to delay such changes and plan for them.

Reply:
so it's not clear for me - are you fine with `part`/`part of` in this PR? this doesn't change file structure

Reply:
That involves putting something in a separate file, right?

Reply:
Removing `part of` and `utils.dart` it's fine.

Reply:
done